### ISSUE TITLE
Funky Proofreader behavior on iPad

### ISSUE THREAD

**Issue by emilia-friedberg** [empirical-org/Empirical-Core#5079]
**Describe the bug**
1. When deleting text on Proofreader on an iPad, the text editor will occasionally act as though the user has held the backspace button and start deleting more than the last character or selected text.

**To Reproduce**
1. Play a Proofreader activity on iPad and try to delete something (may have to hunt for a while before able to replicate).

**Expected behavior**
We should only be deleting the last character or the selected text.

**Impact**
This impacts students who play Proofreader activities on an iPad.

**Screenshots**

**Desktop (please complete the following information):**
 - OS
- Chrome and Safari

**Smartphone (please complete the following information):**

**Additional context**

**Comment by emilia-friedberg** [comment]
We have not actually received user reports of this bug - instead, I discovered it while testing on an iPad. However, after adding a lot of logging functions to the code to test it, I realized that the code appears to be working correctly and is merely registering many Backspace events rather than the one I expect it to when a user hits the Backspace button once. I played the activity on my iPhone and did not experience this bug. Further Googling indicated that the Backspace button getting "stuck" is a known iOS bug, which makes me think that's what may be happening in this case rather than a code issue. I'm going to set up Inspectlet for Proofreader so I can investigate whether this is actually happening to more users or not.
